jina/executors/decorators.py

```diff
@@ -226,8 +226,8 @@ def _merge_results_after_batching(
 
 
 def batching(
-    func: Callable[[Any], np.ndarray] = None,
-    batch_size: Union[int, Callable] = None,
+    func: Optional[Callable[[Any], np.ndarray]] = None,
+    batch_size: Union[int, Callable, None] = None,
```

Comment:
This needs to be Optional[Union[int, Callable]]

Reply:
done